doc/DeveloperGuide.md

```diff
@@ -85,6 +85,29 @@ Use case ends.
 
 3a. The given index is invalid
 
+> 3a1. AddressBook shows an error message <br>
+  Use case resumes at step 2
+  
+#### Use case: Rename tag
+
+**MSS**
+
+1. User requests to rename tag
+2. AddressBook shows a list of tags
+3. User requests to rename a specific person in the list
```

Comment:
User requests to rename a specific...person in the list? The MSS can be a little clearer :)